Formula/jboss-forge.rb

```diff
@@ -15,7 +16,6 @@ def install
   end
 
   test do
-    ENV["FORGE_OPTS"] = "-Duser.home=#{testpath}"
```

Comment:
are we sure this is passed through transparently by just setting it in _JAVA_OPTIONS

Reply:
I tested this one & it works with just our magic.